packages/components/app/styles/components/side-nav/main.scss

```diff
@@ -66,56 +72,98 @@
 
 .hds-side-nav__menu-toggle-button {
   position: absolute;
+  top: -2px;
+  left: -1px;
   z-index: 1;
+  width: 24px;
+  height: 36px;
+  padding: 9px 0;
   color: var(--token-color-foreground-high-contrast);
   background: none;
-  border: 1px solid transparent;
+  background-color: var(--token-side-nav-color-surface-primary);
+  border: 1px solid var(--token-color-palette-neutral-500);
+  border-left-color: transparent;
+  border-top-right-radius: 5px;
+  border-bottom-right-radius: 5px;
+  transform:
+    translateX(var(--hds-app-sidenav-width-expanded))
+    translateY(24px);
   cursor: pointer;
-  transition-timing-function: var(--hds-app-sidenav-animation-easing);
-  transition-duration: var(--hds-app-sidenav-animation-duration);
-  transition-property: width, height, padding, border-radius, transform;
+  transition:
+    transform var(--hds-app-sidenav-animation-duration)
+    var(--hds-app-sidenav-animation-easing),
+    width var(--hds-app-sidenav-animation-duration)
+    var(--hds-app-sidenav-animation-easing);
+
+  &::before {
```

Comment:
[MEMO] discuss if this could be something else (an SVG?)

Reply:
Discussed, this can be an option to explore if we want.

Reply:
We both did a quick test, and the SVG option is not easy at all:
- it needs SVG SMIL animation (in theory, deprecated in favour of CSS animation)
    - it may not be hardware accelerated (depends on multiple factors, the documentation on the web is not very clear on this)
- it's hard to coordinate animations/transitions with `mousover/mouseleave` and get them in sync with other CSS-based animations
- it may be complex to maintain in the future, if someone work on this code without knowing the details

So we have decided to leave it as "plan B" in case "plan A" finds unexpected roadblocks.